collect_app/build.gradle

```diff
@@ -350,7 +350,7 @@ dependencies {
     testImplementation "org.mockito:mockito-core:3.6.28"
 
     testImplementation project(path: ':testshared')
-    testImplementation "org.robolectric:robolectric:4.4"
+    testImplementation "org.robolectric:robolectric:4.3.1"
```

Comment:
Robolectric 4.4 was a massive release and a lot of our tests would fail. This is a good candidate for a separate pr so I'll file a separate issue.